examples/pod_exec_terminal_size.rs

```diff
@@ -0,0 +1,119 @@
+use futures::{channel::mpsc::Sender, SinkExt, StreamExt};
+use k8s_openapi::api::core::v1::Pod;
+
+use kube::{
+    api::{
+        Api, AttachParams, AttachedProcess, DeleteParams, PostParams, ResourceExt, TerminalSize,
+    },
+    Client, runtime::wait::{await_condition, conditions::is_pod_running},
+};
+use tokio::{io::AsyncWriteExt, select};
+
+// send the new terminal size to channel when it change
+async fn handle_terminal_size(mut channel: Sender<TerminalSize>) {
+    let (width, height) = crossterm::terminal::size().unwrap();
+    channel
+        .send(TerminalSize { height, width })
+        .await
+        .expect("fail to write new size to channel");
+
+    let mut stream = tokio::signal::unix::signal(
```

Comment:
There's [`crossterm::event::EventStream`](https://docs.rs/crossterm/latest/crossterm/event/struct.EventStream.html) and [`Event::Resize`](https://docs.rs/crossterm/latest/crossterm/event/enum.Event.html#variant.Resize).

`EventStream` example https://github.com/crossterm-rs/crossterm/blob/master/examples/event-stream-tokio.rs#L26

Reply:
I didn't saw it, thanks I'll use it.